### ISSUE TITLE
Associate Helen Chmura's ORCID ID (0000-0001-6626-8534) with current email address

### ISSUE THREAD

**Issue by jleighherzog** [CDL-Dryad/dryad-product-roadmap#1372]
Please associate the author's ORCID ID with the current email address provided below: 

Previous email addresses: hechmura@ucdavis.edu 
Current email addresses: hchmura@alaska.edu

**Comment by sfisher** [comment]
OK.  I updated her email address in her user record.  Looks like her author information was already updated.

I didn't see two separate accounts, so just updated the one account with the correct email address and changed it from UC Davis to Alaska.

**Comment by jleighherzog** [comment]
Thanks, @sfisher. With the affiliation update to University of Alaska Fairbanks, the final page of the submission form still indicates: "Payment for this deposit is sponsored by University of California, Davis." Do you know why that might be?

**Comment by sfisher** [comment]
If this was a dataset that was started under UC Davis then sponsorship doesn't change automatically.  It looks like she started it a month or so ago before the account was changed.  Sponsorship always stays with the dataset once it's started and doesn't move around with users, so when a dataset is made as a sponsored one, then it stays with the sponsor.

If she wasn't at UC Davis when she began this dataset, I'm happy to reset this to a generic Dryad sponsorship which will charge her.  That's maybe more likely what is desired in this case, but in most cases, we shouldn't take away sponsorship of an older item because someone has moved institutions if they did the work under an old institution.

**Comment by sfisher** [comment]
I changed this to Dryad.  If you think it should by sponsored under UC Davis because she was there when she started this dataset and they should "own" it, then let me know.

**Comment by jleighherzog** [comment]
@sfisher: Mariah had corresponded with the author based on your initial reply. Could you kindly revert back to "UC Davis"?

**Comment by sfisher** [comment]
Ok.  It's being sponsored by UC Davis again.

**Comment by jleighherzog** [comment]
@sfisher : Reply from the author: "It would be most appropriate for it to be associated with my current institution (University of Alaska Fairbanks) where the work was done and for us to be billed." Can you kindly make the change again please?

**Comment by jleighherzog** [comment]
@sfisher : Revisiting this ticket. Before we proceed to publish, can you remove the UC Davis sponsorship and instead associate the submission with  Univ of Alaska Fairbanks?

**Comment by sfisher** [comment]
Hi @jleighherzog I looked at this dataset ( https://datadryad.org/stash/dataset/doi:10.25338/B8QD1H ).  As far as I can tell, it all looks OK for the author showing from University of Alaska Fairbanks.  It will also appear in search this way (with all authors institutions rather than anything special for submitter institution).

The short version:  I think you are fine going ahead publishing this dataset.  Then let us know after it's published and we can manually reset the payment info and fiddle with something to change the payment and send out the invoice.  I don't think the invoice shows institution at all, but just name and dataset information.  I talked with Ryan about this since he's more familiar with invoicing and he thinks it will work.

---
The TLDR:

We can't move the original submitter institution since most UCs use EZID for their DOIs while everyone else uses DataCite.  EZID can't update the DataCite DOIs and DataCite can't update the EZID ones, so it will make updates fail if we move it (and the DOI is supposed to be permanent and unchanging).

The UCs doing their own special EZID thing the issue with moving it.  Everyone else uses Dryad/DataCite DOIs and if we're moving between non-UC institutions we don't have these problems because they're not "special" like the UCs are.  ;-)
